### ISSUE TITLE
networkinterface.getCarrierIPAddress is not a function

### ISSUE THREAD

**Issue by tundak** [salbahra/cordova-plugin-networkinterface#22]
getting issue `networkinterface.getCarrierIPAddress` is not a function, if i use `networkinterface.getIPAddress` this only return IP for wifi, if device using mobile data than return ip undefined.

**Comment by dharders** [comment]
Hey @tundak 

I just stumbled across this too. Simple fix is:

`cordova plugin remove cordova-plugin-networkinterface`
`cordova plugin add https://github.com/salbahra/cordova-plugin-networkinterface`

This will install from master branch (with `getCarrierIPAddress()`) as the latest npm build currently does not have this functionality, even though the docs say it does. 

I've created a new issue #23 so new users don't experience this too, but for now we'll have to use git master as above until the owner @salbahra publishes v1.10.0 to npm.
